examples/speechbrain_tutorial/mini_librispeech_prepare.py

```diff
@@ -0,0 +1,186 @@
+"""
+Downloads and creates manifest files for speech recognition with Mini LibriSpeech.
```

Comment:
There should be a link to the original version, and perhaps a short explanation of the changes done.

Reply:
There is no change here, so I will add the link to the original version.

Reply:
Fixed in 3c801219f00ff9ee3776b8ff4e3bcc20964ad513, thanks!